proxy_multi_map.go

```diff
@@ -206,7 +221,7 @@ func (m *MultiMap) ForceUnlock(ctx context.Context, key interface{}) error {
 	}
 }
 
-// Get returns the value for the specified key, or nil if this multi-map does not contain this key.
+// Get returns the values for the specified key, or an empty slice if this  multi-map does not contain this key.
```

Comment:
How about: "Get returns values for the specified key, ..."?

Reply:
Fixed, [4df6cf3](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/4df6cf34201606cf13235de430fa990af177b711)